website/community/Working-Groups.mdx

```diff
@@ -0,0 +1,88 @@
+
+-   Creating user-story based end-user documentation that to simplify complex concepts for better understanding.   
+-   Creating modular content that can be reused/repurposed for different formats and different use cases.  
+-   Adhering to minimalism practices, logically structuring our content, and endeavouring to make it as findable as possible.
+-   Reduce the number of developer/user issues the engineering team needs to deal with on GitHub.
+
+As part of the Documentation Working group, we work asynchronously with the engineering teams to set clear Release, feature freeze, and code freeze dates, which allows us to set our Documentation freeze and even Release Notes dates. Remember, [docs or it didn’t happen](https://opensource.com/business/15/8/docs-or-it-didnt-happen).
+
+The areas we work on are:
+
+-   **Training material**: Moja global’s training material helps contributors upskill quickly and makes them fit to contribute in a short space of time. We publish our training materials over Moodle learning management system (LMS) which will be expanded over time.    
+-   **Website maintenance**: We aid in the management and maintenance of our main website and the community website, working around updating content, writing blogs and content which can be published centrally.    
+-   **Promotion**: We work on promotional content for social media channels that can be utilized to spread the word about moja global and how people can contribute to our initiative.
+-   **Coordination**: Our contributors put a lot of focus on a documentative approach to communication. Every message is written to be reused or referenced in the future. The coordination effort relies on finding these resources which can be integrated with the documentation and coordinating the forums and mailing lists.
+
+We are yet to decide on the tools and technologies that we will employ for our purpose. However some of them that are most commonly employed are [Markdown](https://guides.github.com/features/mastering-markdown/), [Docusaurus](https://docusaurus.io/), [Sphinx](https://www.sphinx-doc.org/), [ReadTheDocs](https://readthedocs.org/), [Moodle](https://moodle.org/), [GitHub Issues](https://guides.github.com/features/issues/), [GitHub Project Board](https://github.com/features/project-management/). We also would like to use more efficient project management tools to track our documentation and releases and we are open to suggestions for the same.
+
+All the Contributors for the Documentation Working Group are invited to a bi-weekly sync call to discuss the completed, ongoing and pending work. If you would like to join as a volunteer, please fill-up the Google form: [**https://forms.gle/ibH2XsaBSAeiyQcX7**](https://forms.gle/ibH2XsaBSAeiyQcX7)
```

Comment:
Biweekly -> fortnightly. Biweekly is ambiguous and can mean twice per week because English sucks.